### DISCUSSION TITLE
哔哩打开蓝屏，加载不出界面

### DISCUSSION THREAD

**Discussion by tianguoyan** [Richasy/Bili.Uwp#796] [问答] [answered]
如图
![image](https://user-images.githubusercontent.com/18096129/156902940-71ad1b5e-313a-42f8-81ed-b0144500d604.png)

**Comment by tianguoyan** [comment, marked as answer]
更新系统至21H2(19044.1288)，问题就解决啦。已实测ltsb2015、2016、ltsc2019均不可使用此软件。其中ltsb2015无法安装，ltsb2016、ltsc2019可以安装，打开蓝屏。
从ltsb2015一步步升级到ltsc2021，感觉系统资源占用提升很明显，但不是线性的，2016最最吃资源，19、21反而好些，21比19还好，最省资源是15。

**Reply by DeyuanKong025** [reply]
您好，我也遇到了同样的问题，如果不想更新系统的话下载哔哩以前的版本可以吗
